src/Database/DBStructure.php

```diff
@@ -203,8 +203,8 @@ private static function printUpdateError($message) {
 	 * @param array $definition An array of the definition tables
 	 * @return string Empty string if the update is successful, error messages otherwise
 	 */
-	public static function update($verbose, $action, array $tables = null, array $definition = null) {
-		if ($action) {
+	public static function update($verbose, $action, $install = false, array $tables = null, array $definition = null) {
```

Comment:
Please add the `$install` parameter to the doxygen header

Reply:
Oops